widgets/contact/fields/radio.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class SiteOrigin_Widget_ContactForm_Field_Radio extends SiteOrigin_Widget_ContactForm_Field_Base {
+
+	public function render_field( $options ) {
+		if ( ! empty( $options['field']['options'] ) ): ?>
+		<ul>
+			<?php foreach ( $options['field']['options'] as $i => $option ): ?>
+			<li>
+				<label>
```

Comment:
Funny story - I actually used the checkbox field as a base and it turns out that has a missing label. :S I've updated the checkboxes structure - how does that look?